reconcile/acs_notifiers.py

```diff
@@ -0,0 +1,157 @@
+from reconcile.utils import gql
+from reconcile.utils.acs.notifiers import (
+    AcsNotifiersApi,
+    JiraCredentials,
+    JiraNotifier,
+    SeverityPriorityMapping,
+)
+from reconcile.utils.differ import diff_iterables
+from reconcile.utils.disabled_integrations import integration_is_enabled
+from reconcile.utils.runtime.integration import (
+    NoParams,
+    QontractReconcileIntegration,
+)
+from reconcile.utils.secret_reader import SecretReaderBase, create_secret_reader
+from reconcile.utils.semver_helper import make_semver
+
+
+class AcsNotifiersIntegration(QontractReconcileIntegration[NoParams]):
+    def __init__(self) -> None:
+        super().__init__(NoParams())
+        self.qontract_integration = "acs-notifiers"
+        self.qontract_integration_version = make_semver(0, 1, 0)
+        self.gql_api = gql.get_api()
+
+    @property
+    def name(self) -> str:
+        return self.qontract_integration
+
+    def _get_escalation_policies(self) -> list[gql_acs_policies.AppEscalationPolicyV1]:
+        return list(
+            {
+                p.integrations.notifiers.jira.escalation_policy.name: p.integrations.notifiers.jira.escalation_policy
+                for p in gql_acs_policies.query(
+                    query_func=self.gql_api.query
+                ).acs_policies
+                or []
+                if p.integrations
+                and p.integrations.notifiers
+                and p.integrations.notifiers.jira
+                and integration_is_enabled(
+                    self.qontract_integration,
+                    p.integrations.notifiers.jira.escalation_policy.channels.jira_board[
+                        0
+                    ],
+                )
+            }.values()
+        )
+
+    def _build_jira_notifier(
+        self, escalation_policy: gql_acs_policies.AppEscalationPolicyV1
+    ) -> JiraNotifier:
+        jira_board = escalation_policy.channels.jira_board[0]
+
+        custom_fields: dict[str, Any] = {}
+        if jira_board.issue_security_id:
+            custom_fields["security"] = {"id": jira_board.issue_security_id}
+        if escalation_policy.channels.jira_component:
+            custom_fields["components"] = [
+                {"name": escalation_policy.channels.jira_component}
+            ]
+        if escalation_policy.channels.jira_labels:
+            custom_fields["labels"] = escalation_policy.channels.jira_labels
+
+        item = JiraNotifier(
+            name=f"jira-{escalation_policy.name}",
+            board=jira_board.name,
+            url=jira_board.server.server_url,
+            issue_type=jira_board.issue_type or "Task",
+            severity_priority_mappings=sorted(
+                [
+                    SeverityPriorityMapping(**vars(sp))
+                    for sp in jira_board.severity_priority_mappings.mappings
+                ],
+                key=lambda m: m.severity,
+            ),
+            custom_fields=custom_fields if custom_fields else {},
```

Comment:
```suggestion
            custom_fields=custom_fields,
```
should be enough

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4246/commits/3785a1894470470b14165eea38537f9a4c5440f4